.storybook/recipes/Sliders.stories.tsx

```diff
@@ -0,0 +1,230 @@
+import type { StoryObj, Meta } from '@storybook/react';
+// import { userEvent } from '@storybook/testing-library';
+import React, { useState } from 'react';
+import { Slider, InputField, Text, Button, Label } from '../../src';
+
+export default {
+  title: 'Recipes/Sliders',
+  component: Slider,
+  decorators: [
+    (Story) => (
+      <div className="p-8">
+        <Story />
+      </div>
+    ),
+  ],
+  argTypes: {
+    fieldNote: {
+      type: 'string',
+    },
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Slider>;
+
+const moodData = [
+  { emoji: '😡', value: 0, description: 'Very Upset' },
+  { emoji: '🙁', value: 25, description: 'Upset' },
+  { emoji: '😐', value: 50, description: 'Neutral' },
+  { emoji: '🙂', value: 75, description: 'Happy' },
+  { emoji: '😍', value: 100, description: 'Very Happy' },
+];
+
+export const UsingInputDisplay: StoryObj<Args> = {
+  parameters: {
+    axe: {
+      disabledRules: ['color-contrast'], // adding for disabled field example
+    },
+  },
+  render: ({ min = 0, max = 100, step = 1, value = 50, ...rest }) => {
+    /* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
good callout. the IDE suggested this change, and i didn't scrutinize it since it made squiggly go away (my main objection to overusing AI assistants in a nutshell :P) 

Since this has quite a few tests, and could have more, i'll put a single one at top explaining why its there